.github/workflows/build-and-push.yaml

```diff
@@ -15,7 +15,7 @@ env:
   IMAGE_NAME: mzcld-demo
   LOAD_TESTS_IMAGE_NAME: mzcld-demo-load-tests
   GCP_PROJECT_ID: moz-fx-mzcld-demo-prod
-  IMAGE_NAMESPACE: us-docker.pkg.dev/moz-fx-mzcld-demo-prod/mzcld-demo-prod
+  IMAGE_REPO_PATH: moz-fx-mzcld-demo-prod/mzcld-demo-prod
```

Comment:
This value gets passed in as `image_repo_path` and gets appended to the repository host in the docker-push action.